client/src/components/Standups/StandupList.tsx

```diff
@@ -15,8 +15,11 @@ const StandupList = (props: ListProps) => (
     <Datagrid>
       <TextField source="id" />
       <TextField source="title" />
-      <TextField source="channel" />
-
+      <ArrayField source="channel">
+        <SingleFieldList>
+          <ChipField source="slackChannelId" />
```

Comment:
We need to display channel name
```suggestion
          <ChipField source="name" />
```

Reply:
was thinking of doing it in a different branch with changes for SelectInput as well, needed to read values of dummy data in this one

Reply:
Okay